src/main/java/it/gov/pagopa/cgn/portal/model/DiscountBucketCodeSummaryEntity.java

```diff
@@ -0,0 +1,48 @@
+package it.gov.pagopa.cgn.portal.model;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.EqualsAndHashCode.Exclude;
+import lombok.ToString;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+
+@Entity
+@Table(name = "discount_bucket_code_summary")
+@Data
+public class DiscountBucketCodeSummaryEntity implements Serializable {
+
+    @Id
+    @Column(name = "discount_bucket_code_summary_k")
+    @SequenceGenerator(name = "discount_bucket_code_summary_k_seq", sequenceName = "discount_bucket_code_summary_k_seq", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "discount_bucket_code_summary_k_seq")
+    @Exclude
+    @ToString.Exclude
+    private Long id;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
```

Comment:
why excluding discount form Hashcode?

Reply:
You're right, we should allow that.

Reply:
I left Exclude because it's usually put on all relations. https://github.com/pagopa/cgn-onboarding-portal-backend/pull/107/commits/9e9779ea784ca535665776493e2eb2b14a7bb3b3